packages/experimental/Avatar/src/JSAvatarTokens.win32.ts

```diff
@@ -0,0 +1,272 @@
+      fontSize: globalTokens.font.size[900],
+      square: {
+        borderRadius: globalTokens.corner.radius.extraLarge,
+      },
+    },
+    neutral: {
+      backgroundColor: t.colors.neutralBackground6,
+      color: t.colors.neutralForeground3,
+      ringColor: t.colors.transparentStroke,
+    },
+    brand: {
+      backgroundColor: t.colors.brandBackgroundStatic,
+      color: t.colors.neutralForegroundOnBrand,
+      ringColor: t.colors.transparentStroke,
+    },
+    darkRed: {
+      ...getColorProps('darkRed', t),
+    },
+    cranberry: {
+      ...getColorProps('cranberry', t),
+    },
+    red: {
+      ...getColorProps('red', t),
+    },
+    pumpkin: {
+      ...getColorProps('pumpkin', t),
+    },
+    peach: {
+      ...getColorProps('peach', t),
+    },
+    marigold: {
+      ...getColorProps('marigold', t),
+    },
+    gold: {
+      ...getColorProps('gold', t),
+    },
+    brass: {
+      ...getColorProps('brass', t),
+    },
+    brown: {
+      ...getColorProps('brown', t),
+    },
+    forest: {
+      ...getColorProps('forest', t),
+    },
+    seafoam: {
+      ...getColorProps('seafoam', t),
+    },
+    darkGreen: {
+      ...getColorProps('darkGreen', t),
+    },
+    lightTeal: {
+      ...getColorProps('lightTeal', t),
+    },
+    teal: {
+      ...getColorProps('teal', t),
+    },
+    steel: {
+      ...getColorProps('steel', t),
+    },
+    blue: {
+      ...getColorProps('blue', t),
+    },
+    royalBlue: {
+      ...getColorProps('royalBlue', t),
+    },
+    cornflower: {
+      ...getColorProps('cornflower', t),
+    },
+    navy: {
+      ...getColorProps('navy', t),
+    },
+    lavender: {
+      ...getColorProps('lavender', t),
+    },
+    purple: {
+      ...getColorProps('purple', t),
+    },
+    grape: {
+      ...getColorProps('grape', t),
+    },
+    lilac: {
+      ...getColorProps('lilac', t),
+    },
+    pink: {
+      ...getColorProps('pink', t),
+    },
+    magenta: {
+      ...getColorProps('magenta', t),
+    },
+    plum: {
+      ...getColorProps('plum', t),
+    },
+    beige: {
+      ...getColorProps('beige', t),
+    },
+    mink: {
+      ...getColorProps('mink', t),
+    },
+    platinum: {
+      ...getColorProps('platinum', t),
+    },
+    anchor: {
+      ...getColorProps('anchor', t),
+    },
+  } as JSAvatarTokens);
+
+/**
+ * A function which returns object of props depending on color and theme.
+ * @param color
+ * @param theme
+ * @returns object of props - backgroundColor, color and ringColor
+ */
+function getColorProps(color: string, theme: Theme) {
+  const themeAppearance = theme.name;
+  switch (themeAppearance) {
+    case 'White':
+    case 'Colorful':
+    default:
+      return {
+        backgroundColor: globalTokens.color[color].tint40,
+        color: globalTokens.color[color].shade30,
+        iconColor: globalTokens.color[color].shade30,
+        ringColor: globalTokens.color[color].primary,
+      };
+    case 'Black':
```

Comment:
>   case 'Black': [](http://example.com/codeflow?start=1&length=16)

There should be a case here for 'DarkGray' as well which follows the dark theme colors.

Reply:
Fixed. Thank you